### PR TITLE
Announcement for Fine-3 release

### PR DESCRIPTION
**Pull request by JPrause**
For when we have completed smoke testing Fine-3

### CODE REVIEW

**Comment by JPrause** [comment]
@simaishi with Carol out, would you be able to review.

**Comment by simaishi** [comment]
looks ok to me...

File: site/_posts/2017-08-03-Announcing-Fine-3.html.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+date: 2017-08-03 08:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We've just built Fine-3. This release contains security fixes, bug fixes, UI tweaks, and stabilization.
+
+- Security Fixes:
+  * [CVE-2016-7047](https://access.redhat.com/security/cve/CVE-2016-7047) - A flaw was found in the CloudForms API. A user with permissions to use the MiqReportResults capability within the API could potentially view data from other tenants or groups to which they should not have access.
```
Comment (chessbyte) [inline site/_posts/2017-08-03-Announcing-Fine-3.html.md]: CloudForms?  Suggest "A flaw was found in the REST API"

File: site/_posts/2017-08-03-Announcing-Fine-3.html.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  * [CVE-2017-2664](https://access.redhat.com/security/cve/CVE-2017-2664) - CloudForms lacks RBAC controls on certain methods in the rails application portion of CloudForms. An attacker with access could use a variety of methods within the rails applications portion of CloudForms to escalate privileges.
```
Comment (chessbyte) [inline site/_posts/2017-08-03-Announcing-Fine-3.html.md]: Please re-phrase without CloudForms

File: site/_posts/2017-08-03-Announcing-Fine-3.html.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  * [CVE-2017-7497](https://access.redhat.com/security/cve/CVE-2017-7497) - The dialog for creating cloud volumes (cinder provider) in CloudForms does not filter cloud tenants by user. An attacker with the ability to create storage volumes could use this to create storage volumes for any other tenant.
```
Comment (chessbyte) [inline site/_posts/2017-08-03-Announcing-Fine-3.html.md]: Please re-phrase without CloudForms

**Pull request merged**